src/views/Graph.vue

```diff
@@ -46,6 +46,7 @@ import { mixin } from '@/mixins/index'
 const DATA_URL = 'simple-cytoscape-dot.7.js'
 let cy = {}
 let ur = {}
+const config = {}
```

Comment:
I think if we remove this `config` now, and the reference in `data`, it should still work? Looks like the change from line #21 was made this variable unnecessary?

Reply:
yes agreed :-)